src/style/mod.rs

```diff
@@ -151,18 +174,23 @@ pub struct Style {
 
     // Flexbox properies
```

Comment:
All alignment properties (including `gap`) probably ought to be cfg'd on "flexbox OR grid"

Reply:
Should be fixed now, but let me know if there are other properties that also should be gated